hooks/useScale.ts

```diff
@@ -1,13 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT-0
 
-import {create} from 'react-native-pixel-perfect';
+import { create } from 'react-native-pixel-perfect';
```

Comment:
There are 2 files which you have linted. Please add commands in the package.json for linting and run the command for whole project.